src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -2981,14 +2982,22 @@ export default class EthereumApi implements Api {
    * console.log(storage);
    * ```
    */
+  @assertArgLength(5)
   async debug_storageRangeAt(
     blockHash: DATA,
     transactionIndex: number,
     contractAddress: DATA,
     startKey: DATA,
     maxResult: number
   ): Promise<Ethereum.StorageRangeAtResult<"private">> {
-    return this.#blockchain.storageRangeAt(
+    const blockchain = this.#blockchain;
+    if (blockchain.fallback) {
+      throw new Error(
+        "debug_storageRangeAt is not supported on a forked network. See https://github.com/trufflesuite/ganache/issues/3488 for details."
```

Comment:
see the linked issue for why this has to be there.

Reply:
@MicaiahReid The link was in the `Error`.

Reply:
You guys are so dang smart.